module/VuFind/tests/integration-tests/src/VuFindTest/Mink/LibraryCardsTest.php

```diff
@@ -224,7 +224,7 @@ public function testSwitchingCards(): void
 
         // Switch to the second card; we can't currently use findCssAndSetValue() here because
         // it conflicts with the behavior of jumpMenu.
-        $this->findCss($page, '#library_card')->setValue($card2Value);
+        $this->findCssAndSetValue($page, '#library_card', $card2Value);
```

Comment:
I think we need to revert this change -- see the comment above, indicating that findCssAndSetValue doesn't work here because of the jumpMenu behavior. Maybe we can move this change over to the remove jumpmenu PR but revert it here, since it's causing problems.

Reply:
Oops, I didn't notice that one. I was able to reproduce the exception, so I added an option to skip the verification but continue to use findCssAndSetValue with its exception handling for added security.